xml/System.Runtime/JitInfo.xml

```diff
@@ -60,9 +60,9 @@
         <Parameter Name="currentThread" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="currentThread">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="currentThread">Whether the returned value should be specific to the current thread. Default: false</param>
+        <summary>Get the number of bytes of IL that have been compiled. If <paramref name="currentThread" /> is <see langword="true" />, then this value is scoped to the current thread, otherwise, this is a global value.</summary>
```

Comment:
```suggestion
        <summary>Gets the number of bytes of intermediate language that have been compiled. If <paramref name="currentThread" /> is <see langword="true" />, then this value is scoped to the current thread; otherwise, this is a global value.</summary>
```